flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -271,4 +350,17 @@ private SessionHandle initializeSession() {
         service.configureSession(sessionHandle, dataGenSource, -1);
         return sessionHandle;
     }
+
+    private List<RowData> fetchAllResults(
```

Comment:
This should be extracted as a util method, SqlGatewayServiceITCase also has this method.